components/sqs_queue/variables.tf

```diff
@@ -0,0 +1,20 @@
+# Required variables:
+variable "name" {
+  description = "The name for this queue (usually the application name)."
+}
+
+variable "application" {
+  description = "The application this queue is provisioned for (e.g. 'dosomething-rogue')."
```

Comment:
Again, not a big deal but maybe update this to `dosomething-northstar` since Rogue has been archived.

Reply:
Changed in 961fcd1!